src/helpers/pillpad.ts

```diff
@@ -0,0 +1,25 @@
+import type { PCBSMTPad } from "circuit-json"
+
+export const pillpad = (
+  pn: number | Array<string | number>,
+  x: number,
+  y: number,
+  w: number,
+  h: number,
+  radius: number,
```

Comment:
too many parameters, make it an object (we have a 2 parameter rule, this codebase is bad but it's only going to get worse if we don't enforce the two parameter rule)

https://github.com/tscircuit/handbook/blob/main/guides/code.md#1-two-parameter-rule